model/src/main/java/io/github/ygojson/model/data/CardPrints.java

```diff
@@ -17,19 +14,47 @@
 
 		This model contains the same properties as the Card Model with the inclusion of the print list."""
 )
-@Data
-@NoArgsConstructor
-@SuperBuilder(toBuilder = true)
-@JsonPOJOBuilder
+// TODO: this is not properly ordered on the schema file
+// TODO: investigate how to fix this
```

Comment:
Should add an issue for this. Also, not sure if we really wanna have this as a main model that is also documented anyway as it is a composed one.

Reply:
Documented with https://github.com/ygojson/ygojson-tools/issues/28